shell/android-studio/reicast/src/main/java/com/reicast/emulator/Emulator.java

```diff
@@ -107,6 +107,28 @@ public void loadGameConfiguration(String gameId) {
         JNIdc.bootdisk(mPrefs.getString(pref_bootdisk, bootdisk));
     }
 
+    public void isVGACompatible(String gameId) {
```

Comment:
Same as my other comment.

Reply:
This is as general as it gets. This class extends Application.